domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/casetype/CaseTypeService.java

```diff
@@ -20,4 +20,5 @@
 
     Optional<CaseTypeVersionInformation> findVersionInfoByCaseTypeId(String id);
 
+    String findDefinitiveCaseTypeId(String id);
 }
```

Comment:
Is this a findBy method?  Not entirely sure the purpose of the method yet but I shall read on.

Reply:
@adr1ancho - the implementation class calls `findFirstByReferenceIgnoreCaseOrderByCreatedAtDescIdDesc()` in `CaseTypeRepository`.